mne/io/ctf/ctf.py

```diff
@@ -149,13 +150,13 @@ def _read_segment_file(self, data, idx, fi, start, stop, cals, mult):
         with open(self._filenames[fi], 'rb') as fid:
             for bi in range(len(r_lims)):
                 samp_offset = (bi + trial_start_idx) * si['res4_nsamp']
-                n_read = min(si['n_samp'] - samp_offset, si['block_size'])
+                n_read = min(si['n_samp_tot'] - samp_offset, si['block_size'])
```

Comment:
And I'm not entirely sure this is the correct solution to fix the bug. It almost looks like our python version tries to leave out some samples from the end (see https://github.com/mne-tools/mne-python/blob/master/mne/io/ctf/ctf.py#L214), but at least with our spm_faces dataset this wasn't behaving well. Maybe someone with deeper understanding of CTF format can help.